workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentConfigurations.cs

```diff
@@ -365,23 +365,52 @@ public class DeploymentInfo
         /// <summary>
         ///     The SpatialOS project that the deployment is running in.
         /// </summary>
-        public string ProjectName { get; }
+        public string ProjectName { get; private set; }
 
         /// <summary>
         ///     The name of the deployment.
         /// </summary>
-        public string Name { get; }
+        public string Name { get; private set; }
 
         /// <summary>
         ///     The id of the deployment.
         /// </summary>
-        public string Id { get; }
+        public string Id { get; private set; }
 
-        public DeploymentInfo(string projectName, string name, string id)
+        /// <summary>
+        ///     The start time of the deployment.
+        /// </summary>
+        public string StartTime { get; private set; }
+
+        /// <summary>
+        ///     The region that the deployment is in.
+        /// </summary>
+        public string Region { get; private set; }
```

Comment:
Shouldn't this be an `enum`?

Reply:
Ah nice catch, yes

Reply:
I even have the enum for this 😂

Reply:
Actually its a bit more complex as I can't reuse the existing enum as there's another state that can present here which is "UNKNOWN_REGION". 

Worth making a new enum for this? Given we just display it on the UI as a string?

Reply:
I'm fine with storing it as a string as it's just for visualization

Reply:
Yeah, that's fine as a string if it's purely for display